worker-workflow/src/main/java/com/github/cafdataprocessing/WorkflowWorker.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafdataprocessing;
+
+import com.github.cafdataprocessing.processing.service.client.ApiClient;
+import com.github.cafdataprocessing.processing.service.client.ApiException;
+import com.github.cafdataprocessing.processing.service.client.api.AdminApi;
+import com.github.cafdataprocessing.workflow.constants.WorkflowWorkerConstants;
+import com.github.cafdataprocessing.workflow.transform.WorkflowTransformer;
+import com.github.cafdataprocessing.workflow.transform.WorkflowTransformerException;
+import com.hpe.caf.api.worker.DataStore;
+import com.hpe.caf.api.worker.DataStoreException;
+import com.hpe.caf.worker.document.exceptions.DocumentWorkerTransientException;
+import com.hpe.caf.worker.document.exceptions.PostProcessingFailedException;
+import com.hpe.caf.worker.document.extensibility.DocumentWorker;
+import com.hpe.caf.worker.document.model.*;
```

Comment:
List the classes explicitly rather than using a star-import
http://checkstyle.sourceforge.net/config_imports.html#AvoidStarImport

Reply:
OK.